imager/ImagerProfile/models.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveImagerManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveImagerManager,
+                     self).get_queryset().filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Imager profile that is attached to user on creation"""
+    user = models.OneToOneField(User, related_name='profile')
+
+    profile_picture = models.ImageField(null=True, blank=True,
+                                        upload_to='images')
+
+    phone_number = models.CharField(max_length=20, blank=True)
+
+    birthday = models.DateField(null=True, blank=True)
+
+    phone_privacy = models.BooleanField(default=False)
+    birthday_privacy = models.BooleanField(default=False)
+    picture_privacy = models.BooleanField(default=False)
+    name_privacy = models.BooleanField(default=False)
+    email_privacy = models.BooleanField(default=False)
+
+    objects = models.Manager()
+    active = ActiveImagerManager()
+
+    following = models.ManyToManyField('self',
```

Comment:
declare all fields in one block, not scattered both before and after other class attributes like these custom manager statements.